include/swift/AST/SubstitutionMap.h

```diff
@@ -39,7 +39,7 @@
 namespace swift {
 
 class GenericSignature;
-class CenericEnvironment;
```

Comment:
If this didn't cause problems before, it can just be removed. Unnecessary forward declarations would make a good category of gardening :)